reporter-web-app/src/components/SummaryView.js

```diff
@@ -32,12 +32,12 @@ import LicenseStatsTable from './LicenseStatsTable';
 import RuleViolationsTable from './RuleViolationsTable';
 import {
     getOrtResult,
-    getSummaryDeclaredLicenses,
-    getSummaryDeclaredLicensesChart,
-    getSummaryDeclaredLicensesFilter,
-    getSummaryDetectedLicenses,
-    getSummaryDetectedLicensesChart,
-    getSummaryDetectedLicensesFilter,
+    getSummaryDeclaredLicensesProcessed,
+    getSummaryDeclaredLicensesProcessedChart,
+    getSummaryDeclaredLicensesProcessedFilter,
+    getSummaryDetectedLicensesProcessed,
```

Comment:
FYI, there's no such thing as `detectedLicensesProcessed` in ORT's model, only `declaredLicensesProcessed`. Detected licenses are implicitly always processed. So you might want to fix the wording here, although I just realized that you also already before used the term `detectedLicensesProcessed` in the web-app reporter.

Reply:
Yes I use `detectedLicensesProcessed` for any time when Kotlin or JS code cleans up the original detected licenses from the scanner.  I noticed that detected licenses from EvaluatedModel's statistics also include exceptions therefore I thought it's likely we will filter those out in the near future so only licenses get displayed in the summary.

Reply:
FYI: @tsteenbe the behavior about the statistics you mention might have been changed meanwhile by 2c7538e.

Reply:
> Yes I use `detectedLicensesProcessed` for any time when Kotlin or JS code cleans up the original detected licenses from the scanner.

My point is: At last Kotlin code never does such a cleanup.

Reply:
Yes, I know currently the Kotlin code does not clean up the detected licenses but it should. Just test against the latest on master still license exceptions are reported as licenses.

I propose a `detectedLicensesProcessed` with a `spdxExpression`, `licenses` and `exceptions` so users do not have to deal themselves with `exceptions` but this is handled by ORT's ScanCode wrapper.

Reply:
> I propose a `detectedLicensesProcessed` with a `spdxExpression`, `licenses` and `exceptions`

Ideally, exceptions (without licenses) should not be an entity on their own. @mnonnenmacher's recent work goes exactly into that direction.

> so users do not have to deal themselves with exceptions but this is handled by ORT's ScanCode wrapper.

Here I agree. The problem is that ScanCode reports exception on their own, and currently our wrapper does not associate these with the belonging license.